extensions/notebook/src/book/tocEntryPathHandler.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import { FileExtension } from '../common/utils';
+
+export class TocEntryPathHandler {
+	public fileInTocEntry: string;
```

Comment:
Can these be public readonly? And then when the path is updated create a new one instead? Right now if one of these is updated there's no guarantee the others are as well which isn't great.